packages/backend-cli/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "backend-cli",
+  "version": "1.0.0",
```

Comment:
Versioning all the packages as the same may be nice. Will they release at different rates?

Reply:
We will use lerna publish to push them out of the door at the same time, with same version number.